src/libguestfish.sh

```diff
@@ -78,4 +78,6 @@ coreos_gf_run_mount() {
 coreos_gf_shutdown() {
     coreos_gf umount-all
     coreos_gf exit
+    GUESTFISH_RUNNING=
```

Comment:
I think this should be `unset GUESTFISH_RUNNING` rather; no reason to leave an empty environment variable around.

Reply:
It's set to empty when this file is first sourced, no?

Reply:
Ah right...I was just copying how the vars were init'ed...my mind is foggy today  :(

Reply:
@jlebon is correct.  Using `unset` causes `unbound variable` errors when the `coreos_gf_run` function is used the second time.

Re-pushing to the original version of the PR.

Reply:
> (So in fact, I think that would break coreos_gf_run as is due to set -u).

Oh hm right.